erpnext/controllers/queries.py

```diff
@@ -634,3 +634,31 @@ def get_fields(doctype, fields=[]):
 		fields.insert(1, meta.title_field.strip())
 
 	return unique(fields)
+
+
+@frappe.whitelist()
+@frappe.validate_and_sanitize_search_inputs
+def item_supplier_query(doctype, txt, searchfield, start, page_len, filters):
+	item_suppliers = frappe.get_all('Item Supplier',
+		filters={'supplier': filters.get('supplier')},
```

Comment:
Shouldn't it also be filtered by the inputs? `txt` and `searchfield`? 

I'm actually not sure how those even work. Some functions in this file use them, and some don't?

Reply:
Okay, it'll pick up the context now

![supplier-item-query](https://user-images.githubusercontent.com/13396535/100443770-5148fa00-30d0-11eb-85bf-b36b0eb3d72d.gif)